prog/aws/instance.rb

```diff
@@ -1,10 +1,72 @@
 # frozen_string_literal: true
 
-require "aws-sdk-ec2"
 class Prog::Aws::Instance < Prog::Base
   subject_is :vm
 
   label def start
+    assume_role_policy = {
+      Version: "2012-10-17",
+      Statement: [
+        {
+          Effect: "Allow",
+          Principal: {Service: "ec2.amazonaws.com"},
+          Action: "sts:AssumeRole"
+        }
+      ]
+    }.to_json
```

Comment:
A bit puzzled here, where's the policy fluff to restrict the ability of the role to only a certain log group?

Reply:
I have updated it, please check again.